libcst/matchers/tests/test_findall.py

```diff
@@ -162,3 +162,34 @@ def foo(bar: int) -> bool:
         visitor = TestTransformer()
         wrapper.visit(visitor)
         self.assertNodeSequenceEqual(visitor.results, [cst.Name("a"), cst.Name("b")])
+
+
+class MatchersExtractAllTest(UnitTest):
+    def assertNodeSequenceEqual(
+        self,
+        seq1: Sequence[cst.CSTNode],
+        seq2: Sequence[cst.CSTNode],
+        msg: Optional[str] = None,
+    ) -> None:
+        suffix = "" if msg is None else f"\n{msg}"
+        if len(seq1) != len(seq2):
+            raise AssertionError(
+                f"\n{seq1!r}\nis not deeply equal to \n{seq2!r}{suffix}"
+            )
+
+        for node1, node2 in zip(seq1, seq2):
+            if not node1.deep_equals(node2):
+                raise AssertionError(
+                    f"\n{seq1!r}\nis not deeply equal to \n{seq2!r}{suffix}"
+                )
+
```

Comment:
Can we have some tests to cover this part?

Reply:
You want tests that cover the tests?

Reply:
I know it's test helper, just feel like we probably don't want to add code that don't run but this can be used to guard future test cases.
This code block is pretty much the same as MatchersFindAllTest, can we just reuse it? E.g. keep those matcher test helpers in a base class.

https://github.com/Instagram/LibCST/blob/675f8b3e8297b362f4de9b1ba6354e35c0fb049f/libcst/matchers/tests/test_findall.py#L25-L35

Reply:
Its unit test code though. Its not used in production, and if a new test is written that fails, it hits this code. I see no value in testing tests nor trying to commonize this code. Its a waste of engineering effort for no benefit.

Reply:
Jimmy, this is dead code. I just checked and its not even called. Why are you focusing on test coverage to the detriment of an actual review?

Reply:
In general, duplicated code adds tech debt and maintenance burden for future. I didn't specific focus on test coverage. Code quality is as important as implemented features to me and I review both of them.

Reply:
You're completely missing the point. I checked the code and it was entirely unused and I deleted it, it looks like a copy-paste mistake. You're focusing on the minutia of "We should have tested code" instead of the bigger picture. This isn't valuable feedback at all.